process/smartContract/hooks/blockChainHook.go

```diff
@@ -661,7 +661,7 @@ func (bh *BlockChainHookImpl) RevertToSnapshot(snapshot int) error {
 // EpochConfirmed is called whenever a new epoch is confirmed
 func (bh *BlockChainHookImpl) EpochConfirmed(epoch uint32, _ uint64) {
 	bh.flagSaveAccountsIfError.Toggle(epoch < bh.saveAccountsIfErrorDisableEpoch)
-	log.Debug("blockchain hook: built in functions save accounts if error", "enabled", bh.flagSaveAccountsIfError.IsSet())
+	log.Debug("blockchain hook: built in functions save accounts always", "enabled", bh.flagSaveAccountsIfError.IsSet())
```

Comment:
"if error" was ok, as this flag is used only to save in error case as long as it is enabled

Reply:
no it was not - as it appeared at every epoch change on log.